src/objecttypes/api/v2/openapi.yaml

```diff
@@ -0,0 +1,715 @@
+openapi: 3.0.3
+info:
+  title: Objecttypes API
+  version: v1
```

Comment:
This should be v2 at the least :) But see comment above: I rather have an exact version.

Reply:
fixed, thanks for catching